src/components/dashboard/GapByRace.vue

```diff
@@ -1,26 +1,74 @@
 <template>
-  <div class="gap-container">
-    <div>
-      <GapChart
-        :active-stats="activeStats"
-        :expected="expected"
-        :field-data="fieldData"
-      />
+  <div class="outer-container">
+    <div
```

Comment:
maybe add a comment above each of the three main chunks with what is happening

Reply:
I added in comments! let me know if there is anything i need to elaborate on !